calico-cloud/multicluster/kubeconfig.mdx

```diff
@@ -53,6 +53,11 @@ Although this example applies remote cluster configurations symmetrically across
 
 ## Before you begin
 
+**OpenShift users**
+
+{{prodname}} supports federated endpoint identity for OpenShift. However, the [federated services feature](services-controller.mdx)
+cannot be used, because OpenShift currently does not allow users to disable the `RestrictedEndpointsAdmission` plugin.
```

Comment:
cannot be used because OpenShift does not currently allow you to disable the `RestrictedEndpointsAdmission` plugin.
```suggestion
cannot be used, because OpenShift currently does not allow users to disable the `RestrictedEndpointsAdmission` plugin.
```